### PR TITLE
MAINT: move aero surface plots and prints

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

- [x] Code maintenance (refactoring, formatting, renaming, tests)

## Pull request checklist

- Code base maintenance (refactoring, formatting, renaming):

  - [x] Docs have been reviewed and added / updated if needed
  - [x] Lint (`black rocketpy`) has passed locally and any fixes were made
  - [x] All tests (`pytest --runslow`) have passed locally

## What is the new behavior?
- Plots and prints related to Aero
- Huge effort of abstracting plots and prints classes, just like the AeroSurface class is currently structured.

## Does this introduce a breaking change?

- [x] Yes (there's no longer a draw() method in the fins' classes)

## Other information

- Notebooks are running as they always were.
- The tests weren't modified, so all the changes happened exclusively in the background. This should make the review much faster.
- This will create several conflicts with the #372 PR, sorry.
- Please check for plural/singular in the name of the new classes that were created.

### CODE REVIEW

File: tests/test_rocket.py [hunk 1]
```diff
@@ -91,8 +91,6 @@ def mainTrigger(p, h, y):
     assert FinSet.allInfo() == None
     # Check if Tail allInfo() is working properly
     assert Tail.allInfo() == None
-    # Check if draw method is working properly
-    assert FinSet.draw() == None
```
Comment (MateusStano) [inline tests/test_rocket.py:95]: Why remove this?

**Review by MateusStano — CHANGES_REQUESTED** [review]

File: rocketpy/plots/aero_surface_plots.py [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+class _TrapezoidalFinsPlots(_FinPlots):
+    """Class that contains all trapezoidal fin plots."""
+
+    def __init__(self, fin_set):
+        super().__init__(fin_set)
+        return None
+
+    def cross_section(self):
+        """Draw the fin shape along with some important information, including
+        the center line, the quarter line and the center of pressure position.
```
[the remark this answers is not in the document]
Comment (Gui-FernandesBR) [reply rocketpy/plots/aero_surface_plots.py:169]: Okay, I'm going to fix it

File: tests/test_rocket.py (hunk 1 above)
Comment (Gui-FernandesBR) [reply tests/test_rocket.py:95]: The Fin.draw() method doesn't exist anymore. Only the Fin.plots.draw()

File: rocketpy/plots/aero_surface_plots.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gui-FernandesBR) [reply rocketpy/plots/aero_surface_plots.py]: Fixed here: [8f3c58c](https://github.com/RocketPy-Team/RocketPy/pull/381/commits/8f3c58c903a9166a1bc09a39114571da00259d7f)

File: tests/test_rocket.py (hunk 1 above)
Comment (Gui-FernandesBR) [reply tests/test_rocket.py:95]: If you really want to, we can create a draw() method inside each of the Fins classes, which will return self.plots.draw(), but I honestly don't see how that could be helpful.

**Comment by Gui-FernandesBR** [comment]
Ready for a re-review @MateusStano

**Review by MateusStano — APPROVED** [review]

**Pull request merged**